src/screens/game_main.java

```diff
@@ -20,6 +20,7 @@ public game_main(GUIConsoleIO cio2) {
 		
 		menu_items = new String[]{"market"
 				,"management"
+				,"station information"
```

Comment:
here or in management that is the question.
don't want to clutter the main menu, but on the other hand i want it to be
accessible in two keystrokes.

Reply:
what do you think?

Reply:
I don't think 6 options is too many. How many more features do we have planned to add that will go on the main menu? If it goes over 9, I'd say move it under management. Otherwise, I think staying on main is fine.

Reply:
ok